### DISCUSSION TITLE
Will it work with my computer? (a Macintosh)

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#632] [General]
I searched the word Macintosh and can't find any mention of it anywhere on this part of the website.

Also I figured a way you could direct connect a phone to a retro blue device.  Connect a charging cable to a USB 3.0 A female-USB C male connector.  The lights on my retro blue device to work on it.

**Comment by kleffa** [comment]
The lights works because it have energy from USB, but usb is not configured

**Comment by SinisterSticks** [comment]
So will it work with a Macintosh if I use Google Chrome as my browser on Macintosh, which one can do?

I want to make sure that it's possible to reconfigure with a Macintosh before I try hooking it up to a Macintosh.

**Reply by kleffa** [reply]
No. Usb have no use for blueretro than programming it

**Reply by SinisterSticks** [reply]
Could I use Macintosh Bluetooth and get a similar effect to PC Bluetooth if I use Google Chrome as the browser interface, which is available on the Macintosh?

**Reply by kleffa** [reply]
For configure it, should work

**Comment by SinisterSticks** [comment]
Just figured out something.  For the USB configure menu to cone up, tge usb plug should not share the same electrical socket as a conputer.

I noticed something similar when using more than one Turtle Beach USB powered headphobe decoder and headset.  If USB are connected by power, they act different than when on separate curcuits

**Comment by SinisterSticks** [comment]
Okay, now I got it paired.    When I try to configure the VB nur retro, Chorome says it's already paired, but it says in order to configure it, I have to "unpaire it"

there is no option to Unpair in the Blueretro.io page.  

1 how does one unpair it?

2.  Let me guess, I set my source to Xbox Adaptive if I want to program buttons if that's my Bluetooth controller.

3.  II understand the default arrangement is good for most games, especially ames with twin joysticks and Normal NES style games, but that assumes you use a Pad.     I don't know how they'll map to a Fight Stick.    How do I have Presets do Hyper Fighting has one setup, and Teleroboxer has another, etc...
4. I notice you sell Game Cube and Wii Classsic hookups, where do I find them?  Also do you sell hookups for Atari 5200, Coleocovision, Intelliviiosn, and Astrocade.    The 5200 uses a DB 15, (a row of 8 and a row of 7). and all the others use DB9 (row of 5 and row of 4).  Also How do I get an external number pad working with them??
5.
